### ISSUE TITLE
Can I use OpenShift/K8s

### ISSUE THREAD

**Issue by rfaircloth-splunk** [splunk/splunk-connect-for-syslog#482]
The following information should be added to the faq. Wordsmith of course

Can't use UDP and TCP on the same port which breaks multiple bluecoat and cisco feeds among others
Long live TCP connections cause well known problems
Layered networking shrinks the maximum UDP message when you can get it to work causes data loss due to truncation and drops
9:00
If this could work it would be great for us but K8s was built for a very different set of problems, syslog is niche enough I would not expect this to change, the only other two protocols this would be an issue for I know of are SNMP and SIP

**Comment by mkarlstrand-splunk** [comment]
How about this? 

Q: Can the SC4S container be deployed using OpenShift or K8s?
A: There are a number of reasons that OpenShift/K8s is not a good fit for syslog, SNMP or SIP.
-  Can't use UDP and TCP on the same port which breaks multiple bluecoat and cisco feeds among others.
- Layered networking shrinks the maximum UDP message which causes data loss due to truncation and drops
- Long lived TCP connections cause well known problems
- OpenShift doesn't actually use Podman, it uses a library to wrap OCI that Podman also uses. this wrapper around the wrapper has some shortcomings that prevent the service definitions SC4S requires.
- Basically, K8s was built for a very different set of problems than syslog

**Comment by rfaircloth-splunk** [comment]
yes

**Comment by mkarlstrand-splunk** [comment]
Change committed
